src/config/feature-flags.ts

```diff
@@ -19,7 +19,6 @@ export enum BooleanFlags {
 	PRIORITIZE_PUSHES = "prioritize-pushes",
 	EXPOSE_QUEUE_METRICS = "expose-queue-metrics",
 	PROCESS_PUSHES_IMMEDIATELY = "process-pushes-immediately",
-	SIMPLER_PROCESSOR = "simpler-processor",
```

Comment:
If we remove this feature flag, please make the default behavior the behavior **with** the for loop (see comment below, as well).

Reply:
we want to remove the for loop though...

There's no reason to have it with graphql.